snippets/gift-card-recipient-form.liquid

```diff
@@ -106,7 +105,6 @@
           {% render 'icon-error' %}
           <span class="error-message">
             {%- if form.errors contains 'email' -%}
-              {{ form.errors.translated_fields.email | capitalize }}
```

Comment:
When there is an error, the messaging is pretty vague.That seem to be because I'm testing locally. 
Shouldn't every store behave the same?
Here is what I'm getting:![](https://screenshot.click/19-27-cvt6g-0msoe.png)

Reply:
So this is an temporary edge case that you should only see if javascript is disabled. That's clearly not the case in this screenshot so very interested in a replication for how you got this.

Reply:
Oh that's fine, the backend that supports the new error messages isn't merged yet. It should look better in the spin links above.

For some more context we originally just copied what the customer form does which is concatenate error messages with their field name in liquid like `{{ name }}{{ message }}` which we knew was bad but we were pressed for time so we didn't mess with the precedent. Now we have some time to revaluate we think it's pretty unacceptable to do this so we're fixing the messages so they are localised properly. That would be a breaking change if not handled properly so we're using the presence of the offset field as a hint the theme can cope with the new better error strings.